src/app/view/components/_common/dot-push-publish-dialog/dot-push-publish-dialog.component.html

```diff
@@ -2,38 +2,129 @@
+                form.get('pushActionSelected').value === 'publish' ||
+                form.get('pushActionSelected').value === 'publishexpire'
+            "
+        >
+            <p-checkbox
+                formControlName="forcePush"
+                label="{{ dotMessageService.get('contenttypes.content.push_publish.force_push') }}"
+                binary="true"
+            ></p-checkbox>
         </div>
 
-        <div class="form__group push-publish-dialog__publish-date" *ngIf="form.get('pushActionSelected').value === 'publish' || form.get('pushActionSelected').value === 'publishexpire'">
-            <label class="form__label">{{ dotMessageService.get('contenttypes.content.push_publish.publish_date') }}: </label>
+        <div
+            class="form__group push-publish-dialog__publish-date"
+            *ngIf="
+                form.get('pushActionSelected').value === 'publish' ||
+                form.get('pushActionSelected').value === 'publishexpire'
+            "
+        >
+            <label class="form__label"
+                >{{ dotMessageService.get('contenttypes.content.push_publish.publish_date') }}:
+            </label>
             <div class="push-publish-dialog__calendar">
-                <p-calendar class="push-publish-dialog__calendar-date" formControlName="publishdate" dataType="string" dateFormat="yy-mm-dd" placeholder="yy-mm-dd" showTime="true" readonlyInput="true" [minDate]="dateFieldMinDate" ></p-calendar>
+                <p-calendar
+                    class="push-publish-dialog__calendar-date"
+                    formControlName="publishdate"
+                    dataType="string"
+                    dateFormat="yy-mm-dd"
+                    placeholder="yy-mm-dd"
+                    showTime="true"
+                    readonlyInput="true"
+                    [minDate]="dateFieldMinDate"
+                ></p-calendar>
             </div>
-            <dot-field-validation-message message="{{ dotMessageService.get('contenttypes.content.push_publish.publish_date_errormsg') }}" [field]="form.get('publishdate')"></dot-field-validation-message>
+            <dot-field-validation-message
+                message="{{
+                    dotMessageService.get('contenttypes.content.push_publish.publish_date_errormsg')
+                }}"
+                [field]="form.get('publishdate')"
+            ></dot-field-validation-message>
         </div>
 
-        <div class="form__group push-publish-dialog__expire-date" *ngIf="form.get('pushActionSelected').value === 'expire' || form.get('pushActionSelected').value === 'publishexpire'">
-            <label class="form__label">{{ dotMessageService.get('contenttypes.content.push_publish.expire_date') }}: </label>
+        <div
+            class="form__group push-publish-dialog__expire-date"
+            *ngIf="
+                form.get('pushActionSelected').value === 'expire' ||
+                form.get('pushActionSelected').value === 'publishexpire'
```

Comment:
I think is better create a method for this or a Observable who listen by the form change, this way the logic is not repeat several times into the template